### ISSUE TITLE
Manual recommends insecure configuration

### ISSUE THREAD

**Issue by lheckemann** [NixOS/nixpkgs#25987]
https://nixos.org/nixpkgs/manual/#sec-steam-play recommends setting

```
services.udev.extraRules = ''
    SUBSYSTEM=="usb", ATTRS{idVendor}=="28de", MODE="0666"
    KERNEL=="uinput", MODE="0660", GROUP="users", OPTIONS+="static_node=uinput"
  '';
```

in order to use the steam controller. This is a bad idea, as it allows all users to inject arbitrary events, which could be used for privilege escalation.

**Comment by bjornfor** [comment]
I haven't found proper documentation, but I've seen others replace MODE/GROUP with something like TAG+="uaccess".

**Comment by fpletz** [comment]
With the `uaccess` tag, udev sets acls for the user on the currently active seat: https://github.com/systemd/systemd/blob/master/src/udev/udev-builtin-uaccess.c

It makes sense and should be safe to use is as a default.

**Comment by lheckemann** [comment]
Does this prevent them from opening it then logging out and leaving their process running with input write access?

**Comment by lheckemann** [comment]
This was fixed in d859a66351181bf63d2f311269e2b71bf949e166, although I'm not sure if steam's udev rules are sensible (will check!).
